content/installation/java/Java-basic-installation.md

```diff
@@ -0,0 +1,20 @@
+<!--
+title: "Java Agent Installation"
+description: "Overview of the Java Agent installation"
+tags: "installation Java agent overview installation"
+-->
+
+To install the Java agent, complete the following steps:
+
+* Log in to the Contrast interface.
+* Click the button in the top navigation bar to **Add Agent**.
+* Select the **Java** agent in the dropdown menu, and click the button to **Download Agent**. You may need to specify proxy authentication information required by your network before downloading the agent.
+
+> **Note:** The **Java** agent can be used with all compatible versions of Java, and will upgrade auto-restart. The **Java 1.5** agent doesn't upgrade auto-restart.
```

Comment:
I feel like "upgrade auto-restart" is a little confusing. What it is doing is automatically upgrading whenever it is re-started.

Reply:
Yeah, our naming of the Java entries in the dropdown is pretty confusing.
The "Java" file isn't the agent. It's a "launcher/updater" that, on start-up, checks TeamServer for the latest version of the Java agent, and downloads that into the user's environment.

The "Java 1.5" file is the actual agent. It will not automatically check to see if there is a newer version of the agent available. 

I really wish we called them something else in the dropdown, instead of having to write documentation to clarify this for users.